mx.py

```diff
@@ -767,7 +767,7 @@ def classpath_repr(self, resolve=True):
         nyi('classpath_repr', self)
 
     def isJar(self):
-        cp_repr = self.classpath_repr()
+        cp_repr = self.classpath_repr() #pylint: disable=assignment-from-no-return
```

Comment:
`self.classpath_repr()` does not have a `return`, but this is because it's throwing a `nyi(...)`. So this is a false positive.

Reply:
Ok.